Service/DeveloperOverheidService.php

```diff
@@ -297,19 +331,88 @@ public function getComponent(string $id): ?array
      *
      * @return ?ObjectEntity
      */
-    public function handleRepositoryArray(array $repository, ?Entity $repositoryEntity = null, ?Source $developerOverheidSource = null): ?ObjectEntity
+    public function handleRepositoryArray(array $repository, ?Source $developerOverheidSource = null): ?ObjectEntity
     {
+        if (!$repositoryEntity = $this->getRepositoryEntity()) {
+            isset($this->io) && $this->io->error('No repositoryEntity found when trying to import a Repository');
+
+            return null;
+        }
+
         // Handle sync
         $synchronization = $this->synchronizationService->findSyncBySource($this->source ?? $developerOverheidSource, $this->repositoryEntity ?? $repositoryEntity, $repository['id']);
         isset($this->io) && $this->io->comment('Checking component '.$repository['name']);
-        $synchronization = $this->synchronizationService->synchronize($synchronization, $repository);
+        $synchronization = $this->synchronizationService->handleSync($synchronization, $repository);
```

Comment:
Is there a reason to revert to handleSync? The synchronize function should replace handleSync in order to force remapping if the mapping changes.